docs/web/concepts-layout.mdx

```diff
@@ -0,0 +1,303 @@
+
+## Slotting
+
+The position of a component within its parent is controlled by what slot it is placed into.
+
+For example, this configuration will place the zoom widget in the top right of the map.
+
+```xml
+<?xml version="1.0" encoding="utf-8" ?>
+<layout xmlns="https://geocortex.com/layout/v1">
+    <map id="myMap">
+        // highlight-next-line
+        <zoom padding="0.5" slot="top-right"/>
+    </map>
+</layout>
+```
+
+If a slot configuration is not given, the component will slot itself into a default position. Each component defines its own slots - not every slot will work for every component.
+
+Slots control how one element fits into another. But how do we control how sibling elements position themselves? The answer is stacks and splits.
+
+## Stacks and Splits
+
+The basic `stack` and `split` components form building blocks that layout authors can use to vertically or horizontally partition components and blocks of rich text to form simple and complex user interfaces alike.
+
+Leveraging these simple primitives correctly also means responsive apps that work on various screen sizes.
+
+Stacks and splits are used to partition the viewer into rectangular sectors, each filled with whichever components you desire.
+
+The best way to think of stacks is that every item within them is being ordered top to bottom:
+
+```xml
+<stack>
+    <text text="item1"/>
+    <text text="item2"/>
+    <text text="item3"/>
+</stack>
+```
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-1.png")} />
+
+The best way to think of splits is that every item within them is being ordered left to right:
+
+```xml
+<split>
+    <text text="item1"/>
+    <text text="item2"/>
+    <text text="item3"/>
+</split>
+```
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-2.png")} />
+
+When creating complex layouts with embedded stacks and splits, it's good to picture building it from the inside out:
+
+```xml
+<stack>
+    <split>
+        <text text="item1"/>
+        <stack>
+            <text text="item2"/>
+            <text text="item3"/>
+        </stack>
+    </split>
+    <split>
+        <text text="item4"/>
+        <text text="item5"/>
+        <text text="item6"/>
+    </split>
+</stack>
+```
+
+We start by building the first inner-most stack:
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-4.png")} />
+
+Then slot that into the split one level up:
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-5.png")} />
+
+We then build the next split:
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-6.png")} />
+
+And finally slot those two splits into the outer-most stack:
+
+<img className="stackSplitImage" src={useBaseUrl("img/stack-split-3.png")} />
+
+## Panels
```

Comment:
@eric-parsons for these starting docs, we wanted to roll with the minimum useful content. These layout / app config docs currently achieve very rough parity with the existing markdown files in the various framework/web repos.  Tabs and Expands would definitely be a useful thing to mention but I'm leaning towards leaving them out and making a ticket to add them, and slowly build and extend this content. Like you said, it might be a quick win to just mention them and link to the component api! once the component API exists. (it doesn't yet.)

Reply:
Sure. To be clear, I don't think that documenting tabs, expands, dialogs, etc. would make sense at this point, nor would the concepts overview be the best place for that. My concern was mainly that while stacks and splits are somewhat special, panel is just a random component, no more or less useful to know about than a bunch of others that have a similar role. It wasn't clear (to me at least) that "others" existed.

All that's needed here I think is something like "These are just a few examples of building blocks that help you organize the layout of your UI. You can also use tabs, expanders, and much more. See our Component Reference for a full listing (coming soon)."

Reply:
Added a little blurb as suggested.